sdk/keyvault/azure.keyvault.secrets.txt

```diff
@@ -0,0 +1,57 @@
+**** Package/Module azure.keyvault.secrets****
+class SecretClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_secret(self, name: str, **kwargs) -> bytes: ...
+	def delete_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_secret(self, name: str, version: str=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
+	def list_deleted_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.DeletedSecret]: ...
+	def list_secret_versions(self, name: str, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def list_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def purge_deleted_secret(self, name: str, **kwargs): ...
```

Comment:
We discussed putting \*deleted\* methods on a different client in .NET would we want to do the same thing for python

Reply:
Probably not for python as there are already sync vs async clients.